### PR TITLE
Cap applications dashboard

### PR DESCRIPTION
**Pull request by NimRegev**

### CODE REVIEW

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cap-applications-dashboard/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cap-applications-dashboard/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cap-applications-dashboard/docs/

File: _docs/deployment/applications-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
+---
+title: "Applications dashboard"
+description: ""
+group: deployment
+toc: true
+---
+
+View, monitor, and analyze deployments across your enterprise in the Applications dashboard. As a one-stop shop for Argo Rollouts and Argo CD in Codefresh, the Applications dashboard delivers on the challenge of keeping track of deployments, whatever the frequency and scale. A wide range of filters, enriched CI and CD information, provide full traceability and visibility to continuous deployments. 
+
+Here's are some insights you can derive from the Applications dashboard:
```
Comment (kostis-codefresh) [inline _docs/deployment/applications-dashboard.md:10]: Include a screenshot of the dashboard here.

File: _docs/deployment/applications-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+Mark applications as favorites, and view them with a click.  
+* Select the {::nomarkdown}<img src="../../../images/icons/icon-mark-favorite.png?display=inline-block">{:/} to the left of the application name to mark it as a favorite. 
+* To view only favorites, on the filters bar, select {::nomarkdown}<img src="../../../images/icons/icon-fav-starred.png?display=inline-block">{:/}.
+
+
+
+#### Deployment type
+Applications are displayed according to their deployment type which can be one of the following:
+* Applications  
+  Standalone applications. For detailed information, see the official documentation on [Applications](https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/#applications){:target="\_blank"}.
```
[the remark this answers is not in the document]
Comment (NimRegev) [reply _docs/deployment/applications-dashboard.md:89]: Actually no. I missed adding that limitation. For AppSet, they have to do it from Argo.

File: _docs/deployment/applications-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+    For detailed information, see the official documentation on [Labels and selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/){:target="\_blank"}.
+
+    {% include
+image.html
+lightbox="true"
+file="/images/applications/app-dashboard-adv-filters.png"
+url="/images/applications/app-dashboard-adv-filters.png"
+alt="Advanced filters in Applications Dashboard"
+caption="Advanced filters in Applications Dashboard"
+max-width="30%"
```
[the remark this answers is not in the document]
Comment (NimRegev) [reply _docs/deployment/applications-dashboard.md:74]: Why would make it bigger in the thumbnail? Thumbnails are meant to save real estate. Anyways they'll have to click to see the image properly.

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cap-applications-dashboard/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cap-applications-dashboard/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cap-applications-dashboard/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cap-applications-dashboard/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cap-applications-dashboard/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cap-applications-dashboard/docs/

**Pull request merged**